### ISSUE TITLE
Artikel werden ohne Freigabe übertragen

### ISSUE THREAD

**Issue by marcmanusch** [plentymarkets/plentymarkets-shopware-connector#280]
#### Actual Behavior

Artikel werden nach SW übertragen, obwohl diese nicht für den Shop freigegeben wurden.

#### Expected Behavior

Der Artikel darf nicht übertragen werden.

#### Steps to Reproduce

Wir nutzen einen extra Mandanten, auch ohne Freigabe wird der Artikel übertragen.

#### Possible Solutions

Ich würde es gut finden, wenn man es so lösen könnte wie bei allen anderen Marktplätzen. Z.B. die Freigabe für den Shop über den Marktplatz "WEB-API" nutzen.

**Also Freigabe für Mandanten --> + WEB-API = Freigabe Shopware**

**Comment by jochenmanz** [comment]
kannst du hier einmal Screenshots von Beispielartikeln posten anhand dessen ich das ganze in einem Testsystem nachstellen kann?

**Comment by marcmanusch** [comment]
Klar...generell ist das problem halt, wie gebe ich Artikel nicht für den Shop frei ? der Mandant "Shopware" ist bei uns für alle Plattformen (ebay,Amazon,etc).
![b1](https://user-images.githubusercontent.com/31078285/38096604-9aaa9742-3373-11e8-933c-0c6b115f608c.jpeg)
![b2](https://user-images.githubusercontent.com/31078285/38096607-9d404254-3373-11e8-8fd8-4bca27332bbf.jpeg)

**Comment by marcmanusch** [comment]
Ja, aber habe momentan verstandes weshalb.... wir haben einen eigenen Mandanten für Shopware, jedoch nutzen wir diesen auch für alles Andere. Also wir nutzen den Standard Mandanten gar nicht.

Ich habe im Plenty Forum den PR gesehen, der auch die Herkunft prüft, jedoch führt das zu großen Problemen bei uns.

Habe es gelöst, indem ich eine Kategorie angelegt habe die es nicht im Shop gibt, sodass der Artikel nicht im Shop angezeigt wird.

Mit freundlichen Grüßen
Marc Manusch
